robotError.py

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
why into another file? u can use OSTools I guess.... 
More files = more uploading time

Reply:
Makes the code more accessible to others and more clear.